sno/ogr_import_source.py

```diff
@@ -387,6 +389,27 @@ def features(self):
         for ogr_feature in self._iter_ogr_features():
             yield self._ogr_feature_to_sno_feature(ogr_feature)
 
+    def get_features(self, row_pks, *, ignore_missing=False):
+        pk_field = self.primary_key
+
+        # TODO: does OGR have any better way of quoting these parameters?
+        # there's no params argument to SetAttributeFilter()...
+        def _quote(x):
+            if isinstance(x, int):
+                return str(x)
+            else:
+                return "'{}'".format(str(x).replace("'", "''"))
+
+        # (123,) --> 123. we only handle one pk field
```

Comment:
assert? Or checked elsewhere?

Reply:
currently there's just no way of getting a multi-PK dataset into sno in the first place, but I'll add an assert anyway